### PR TITLE
Re-optimize and clean up settings, social and logo icons

### LINKED ISSUE #4076: Lots of SVG source files broken since svgo optimization
When wanting to edit some icons, I stumbled upon this issue:
capture du 2017-03-26 18-57-33
capture du 2017-03-26 18-55-42

This seems to be caused by the SVGO optimization: https://github.com/nextcloud/server/pull/1534

While they seem to be correctly displayed, Inkscape has big problems showing them correctly. There’s a reason we use Scour as per https://github.com/nextcloud/server/blob/master/core/img/image-optimization.sh#L10 and not SVGO. ;)

cc @skjnldsv could you check it out? In doubt I’d say your PR needs to be reverted and we should do it via Scour instead – which was already used on most of the icons.

### PR DESCRIPTION
**Pull request by pixelipo**
I've discovered a whole set of icons in settings app that needed to be fixed (mostly I just replaced them with already optimized icons found in **core/img/*** folders).

I've also fixed social icons as well as logo and favicon SVGs. Those last ones are quite critical, so test those carefully.

@nextcloud/designers 

(finally) Fixes #4076

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

File: settings/img/users.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 16 16" height="16" width="16" version="1.1"><path d="m9 1c-1.746 0-3 1.43-3 2.8 0 1.4 0.1 2.4 0.8 3.5 0.224 0.29 0.485 0.35 0.7 0.6 0.135 0.5 0.24 1 0.1 1.5-0.436 0.153-0.85 0.332-1.27 0.5-0.51-0.273-1.1-0.5-1.61-0.7-0.07-0.28-0.02-0.487 0.05-0.75 0.12-0.125 0.23-0.18 0.36-0.3 0.37-0.45 0.39-1.21 0.39-1.75 0-0.8-0.72-1.4-1.5-1.4-0.87 0-1.5 0.72-1.5 1.4h-0.02c0 0.7 0.05 1.2 0.4 1.75 0.1 0.15 0.242 0.175 0.35 0.3 0.0674 0.25 0.121 0.5 0.05 0.75-0.64 0.223-1.244 0.5-1.8 0.8-0.42 0.3-0.233 0.182-0.5 1.15-0.124 0.5 1.3 0.73 2.32 0.81-0.05 0.275-0.12 0.64-0.32 1.34-0.32 1.25 4.353 1.7 6 1.7 2.43 0 6.313-0.456 5.98-1.7-0.52-1.94-0.208-1.71-0.98-2.3-1.09-0.654-2.452-1.167-3.6-1.6-0.15-0.557-0.04-0.97 0.1-1.5 0.235-0.25 0.5-0.36 0.72-0.6 0.69-0.884 0.78-2.424 0.78-3.5 0-1.586-1.43-2.8-3-2.8z"/></svg>
```
Comment (skjnldsv) [inline settings/img/users.svg]: Not convinced with that one. Too small. I prefer the old one :)

Comment (jancborchardt) [reply settings/img/users.svg]: Yeah, this shouldn’t be changed in size. The form is visually not filled enough. :) Can you fix this @pixelipo, then it’s good to go.

**Comment by pixelipo** [comment]
Ok @skjnldsv @jancborchardt  - now that icon fills more space. I didn't revert the old icon since it wasn't using the same "figure" as the other "personal" icons.

I've also updated all the other user-related icons to the same "figure" shape:
```	modified:   core/img/actions/user.svg
	modified:   core/img/categories/social.svg
	modified:   core/img/filetypes/text-vcard.svg
	modified:   settings/img/personal.png
	modified:   settings/img/personal.svg
	modified:   settings/img/users.svg
```

Ready to review and merge :)

**Comment by MorrisJobke** [comment]
> Ready to review and merge :)

There are conflicts :/

**Comment by pixelipo** [comment]
Screwed up with rebase, please go here: #7231

**Pull request closed without merging**